Mathlib/Analysis/Normed/Group/Uniform.lean

```diff
@@ -365,6 +365,27 @@ instance (priority := 100) SeminormedCommGroup.to_uniformGroup : UniformGroup E
 instance (priority := 100) SeminormedCommGroup.toTopologicalGroup : TopologicalGroup E :=
   inferInstance
 
+/-! ### SeparationQuotient -/
+
+namespace SeparationQuotient
+
+@[to_additive instNorm]
+instance instMulNorm : Norm (SeparationQuotient E) where
+  norm := lift Norm.norm fun _ _ h => norm_eq_norm_of_dist_eq_zero' <| h.dist_eq_zero
```

Comment:
```suggestion
  norm := lift Norm.norm fun _ _ h => h.map continuous_norm' |>.eq
```

Then you no longer need `norm_eq_norm_of_dist_eq_zero` and its variations. From a purely Mathlib standpoint I would suggest removing them (people should use the [Inseparable](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Defs/Filter.html#Inseparable) API), unless you see some pedagogical value in them.

Reply:
How about adding `Inseparable.norm_eq_norm`? I think that might be handy for discoverability.

Reply:
Sounds good!